examples/react/react-threejs/src/ui/components/UIContainer.tsx

```diff
@@ -0,0 +1,41 @@
+import { useDojo } from "../../dojo/useDojo";
+import { Button } from "@/components/ui/button";
+import { useComponentValue } from "@dojoengine/react";
+import { Entity } from "@dojoengine/recs";
+import { getEntityIdFromKeys } from "@dojoengine/utils";
+
+interface TProps {}
```

Comment:
is this needed? I think this file can be cleaned

Reply:
Removed this too